test/e2e/kubectl.go

```diff
@@ -833,7 +833,7 @@ var _ = Describe("Kubectl client", func() {
 			image := "nginx"
 
 			By("running the image " + image)
-			runKubectlOrDie("run", rcName, "--image="+image, nsFlag)
+			runKubectlOrDie("run", rcName, "--generator=run-controller/v1", "--image="+image, nsFlag)
```

Comment:
We use this in Origin as run-controller/v1 and reserve run/v1 for running DCs. Should be [run/v1](https://github.com/kubernetes/kubernetes/blob/06996d5af95a8c283f670ffebd09adf9396d82dd/pkg/kubectl/cmd/util/factory.go#L119).